data/quarg/quarg news.txt

```diff
@@ -442,3 +442,65 @@ news "student on quarg planets"
+
+news "ringworld quarg"
+	location
+		government "Quarg"
+		attributes "station"
+	name
+		word
+			"Quarg"
+	message
+		phrase
+			"ringworld quarg phrases" 5
+			"quarg poems on species" 1
+
+phrase "ringworld quarg phrases"
+	word
+		`You see a shorter, younger Quarg, likely a child in Quarg years, being shown around the ringworld by an adult. From the looks of it the parent is teaching it about the ring.`
+		`A Quarg is standing uncomfortably close to the walls, tapping it slowly with both palms. You observe it for a few minutes, until it stops abruptly, and goes about its business like the others.`
+		`Standard activity in the ringworld is stopped by a few dozen Quarg in heavily armored suits, moving in a quick pace and escorting a large, tube-like container which floats to accompany them. Other Quarg are quick to make way for the group, steering a great distance from them.`
+		`Some Quarg scientists are dragging along a whole train of linked lab carts, packed fully with utensils, chemicals, components and reagents. It takes nearly half an hour before you see the last of the carts pass by.`
+		`Hundreds of Quarg are blocking most of one of the largest, widest corridors in the ring. They're all sitting down on the floor, staring down blankly in complete silence, with a scarce few seldom glancing up to the ceiling, only to move their gaze downwards once again. Quarg who witness the scene stop any and all conversation, moving elsewhere.`
+		`One Quarg walks up to you, and stares down into your eyes. It stays there in silence, just looking at you, before finally blinking once and moving along, disappearing into the ring's hallways.`
+		`A Quarg is overseeing the unloading of dozens of ship-sized cargo containers that have arrived in the ring. It seems to mentally count the cargo as it's moved out in perfectly identical sets. After what must've been thousands of cargo tons, it clicks off one of dozens of small dots on a data pad, and begins counting again.`
+		`Two Quarg ships land at the same time a few hangar bays away from one another. They open their hatches, and what must be the ships' entire crews come out. The crew of one ship heads to the other, and vice-versa. The crews each board the other's ship, and depart.`
+		`The Quarg seem to have set up some form of festival in the ring: endless stands stretch out as far as you can see, all displaying variations of the same orange-like fruit, ranging from impossibly smooth to unbelievably wrinkled, and also from sizes as small as a pea to as large as a watermelon. You can't figure what quality they like best.`
+		`You see one Quarg turn a sharp corner in what's a haste by their standards. When you look into the corridor it went to, you see it's a dead end, with the Quarg having inexplicably vanished.`
+
+phrase "quarg poems on species"
```

Comment:
I don't know that much about the Quarg, but them having poetry seems a little strange.